src/misc/fs_utils_win32.cpp

```diff
@@ -256,9 +256,49 @@ void set_dos_file_time(const NativeFileHandle handle, const uint16_t date,
 	SetFileTime(handle, nullptr, nullptr, &write_time);
 }
 
+// Includes a Windows specific hack
+// We move the file to be deleted to a temp file before delete
+// This allows a file with the same name to be created even if there are open file handles
+// See bug report about the game Abuse: https://github.com/dosbox-staging/dosbox-staging/issues/4123
 bool delete_native_file(const std_fs::path& path)
 {
-	return remove(path.string().c_str()) == 0;
+	// Prefix only uses first 3 letters, abbreviate DosBox Staging
+	const wchar_t *prefix = L"DBS";
```

Comment:
How about something more descriptive? Like "DBS-deleted-"?

Reply:
From the Windows documentation, it will only use the first three characters:  https://learn.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-gettempfilenamew

>  The null-terminated prefix string. The function uses up to the first three characters of this string as the prefix of the file name. This string must consist of characters in the OEM-defined character set.

It's not really user visible anyway as the file gets deleted immediately after the move.

Reply:
Yeah I think it's fine. You could make it even uglier and more obvious this is some temp file by using a `$DB` prefix or something, but honestly, I don't think the name matters much.

Reply:
It gives a .tmp extension as well to hopefully make it obvious it's a temp file.  It looks like DBS47BC.tmp where the last 4 digits are hex derived from system time.

Reply:
Yeah, maybe do the `$DB` thing. That makes it even more obvious it's a temp file.

Reply:
Changed to `$DB`